bionty/base/__init__.py

```diff
@@ -1,10 +1,36 @@
-"""Bionty base.
+"""Bionty base is a standalone version of Bionty without lamindb support.
```

Comment:
I'd say: "`bionty.base` is a read-only version of bionty for interfacing public ontologies." Or any other sentence that conveys that this is a "read-only" thing and doesn't leave users wondering "why do I need bionty then?".